Makefile

```diff
@@ -83,7 +83,15 @@ update_assets: update_npm_deps
 	$(DOCKER_PREFIX) python manage.py compress_assets
 	$(DOCKER_PREFIX) python manage.py collectstatic --noinput
 
-initialize_docker: initialize_db update_assets
+initialize_docker:
+	$(DOCKER_PREFIX) python manage.py reset_db --noinput
```

Comment:
+1 it shouldn't be needed as we can run on a running container. This was all done long ago before `exec` existed.